crypto-core/src/main/java/com/palantir/crypto2/cipher/AesCtrCipher.java

```diff
@@ -37,37 +35,39 @@
 public final class AesCtrCipher implements SeekableCipher {
 
     public static final String ALGORITHM = "AES/CTR/NoPadding";
-    static final String PROVIDER = "SunJCE";
     static final String KEY_ALGORITHM = "AES";
     static final int KEY_SIZE = 256;
     static final int BLOCK_SIZE = 16;
     static final int IV_SIZE = 16;
+    private static byte[] out = new byte[BLOCK_SIZE];
```

Comment:
we're never reading this, right? maybe indicate this in the name and/or comment.